frontend/app/src/entities/nodes/relationships/ui/relationship-table/relationship-actions-cell.tsx

```diff
@@ -37,14 +38,35 @@ export function RelationshipActionsCell({
   relationshipLabel,
   relationshipKind,
   relationshipName,
+  relationshipsCount,
 }: ActionsCellProps) {
   const [showPropertiesModal, setShowPropertiesModal] = useState(false);
   const [showEditForm, setShowEditForm] = useState(false);
   const [showDissociateModal, setShowDissociateModal] = useState(false);
-  const { schema } = useSchema(relationshipKind);
+
+  const { schema: parentSchema } = useSchema(parentKind);
+  const { schema: peerSchema } = useSchema(relationshipKind);
+
+  const parentRelationship = parentSchema?.relationships?.find((relationship) => {
+    return relationship.name === relationshipName;
+  });
+
+  const peerRelationship = peerSchema?.relationships?.find((relationship) => {
+    return (
+      relationship.peer === parentKind && relationship.direction === parentRelationship?.direction
+    );
+  });
+
   const isEditAllowed = permission.update.isAllowed;
+  const isDissociateAllowed =
+    parentRelationship?.direction === "bidirectional"
+      ? // If bidirectionnal, check from the peer point of view
+        parentRelationship?.optional && peerRelationship?.optional
+      : // Check if it's optionnal or there is enough peers
+        parentRelationship?.optional || relationshipsCount > (parentRelationship?.min_count ?? 1);
```

Comment:
typo in comments, `nn` => `n`.

Also this deserves to be a utils function with tests.

Checking for direction is not enough. Using demo data, you can use `InfraCircuit`.

`InfraCircuit` has an optional relationship many with `InfraBGPSession`.
`InfraBGPSession` has no relationships with `InfraCircuit`.

So we should be allow to dissociate but we're not.